project/profiles/capacity4more/modules/c4m/user/c4m_user/actions/c4m_user_contact_multiple.action.inc

```diff
@@ -0,0 +1,176 @@
+<?php
+
+/**
+ * @file
+ * Callback & forms related to the user contact multiple.
+ */
+
+/**
+ * Implements hook_action_info().
+ *
+ * A custom action to contact a multiple users with a mail action.
+ * Declares action for Site Admin, Group Admins and Group Owner.
+ */
+function c4m_user_contact_multiple_action_info() {
+  $group = c4m_og_current_group();
+  // If not within OG context, don't declare action.
+  if (!$group) {
+    return array();
+  }
+  // If not Site Admin, Group Admin or Group Owner, don't declare action.
+  if (!_c4m_features_og_members_is_power_user($group)) {
+    return array();
+  }
+
+  return array(
+    'c4m_user_contact_multiple_action' => array(
+      'type' => 'user',
+      'label' => t('Contact'),
+      'configurable' => TRUE,
+      'triggers' => array('any'),
+    ),
+  );
+}
+
+/**
+ * Action form function.
+ *
+ * Create 'Contact Multiple' form.
+ */
+function c4m_user_contact_multiple_action_form(array $context, $form_state) {
+  global $user;
+
+  // If 'Select all items on all pages' is selected.
```

Comment:
@ordavidil 
I didn't find an option to change this label at form.
VBO adds ckeckbox fields at  $form['select_all']['this_page'],  $form['select_all']['all_pages'], but when viewing the form, these fields are not available.